bugsnag-android-core/src/main/java/com/bugsnag/android/SessionTracker.java

```diff
@@ -280,6 +282,7 @@ void flushStoredSession(File storedFile) {
         switch (deliveryStatus) {
             case DELIVERED:
                 sessionStore.deleteStoredFiles(Collections.singletonList(storedFile));
+                logger.d("Sent 1 new session to Bugsnag");
```

Comment:
Will it always be precisely 1 session sent here?

Reply:
yes - `Collections.singletonList()` can only hold one object.

Reply:
D'oh, sorry - I missed that!